src/main/java/org/c4sg/service/impl/OrganizationServiceImpl.java

```diff
@@ -88,4 +106,21 @@ public String getLogoUploadPath(Integer organizationId) {
         return LOGO_UPLOAD.getValue() + File.separator + organizationId + IMAGE.getValue();
     }
 
+	@Override
+	public List<OrganizationDTO> findByUser(Integer userId) {
+		User user = userDAO.findById(userId);
+        requireNonNull(user, "Invalid User Id");
+		List<UserOrganization> userOrganizations = userOrganizationDAO.findByUserId(userId);
+		Collections.sort(userOrganizations, (p1, p2) -> {
```

Comment:
Try to use this in the Lambda expression it will be more readable 
`Comparator.comparing( [add the method here] )`